src/components/views/dialogs/InfoDialog.js

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2015, 2016 OpenMarket Ltd
+Copyright 2017 New Vector Ltd.
```

Comment:
Generally you would claim the copyright yourself in a new file you've created, although given this is based on the QuestionDialog class, perhaps just add your own name to the list.

Reply:
Alright, will do!